src/EnergyPlus/AirflowNetwork/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+cmake_minimum_required (VERSION 2.8)
+
+project(AirflowNetwork)
+
+#INCLUDE_DIRECTORIES( ${CMAKE_CURRENT_SOURCE_DIR} )
+
+# public API headers go in the src/EnergyPlus/public folder
+#INCLUDE_DIRECTORIES( ${CMAKE_CURRENT_SOURCE_DIR}/public )
+
+# Don't reconfigure this file when moving from PROFILE_GENERATE to PROFILE_USE
+#if(NOT PROFILE_USE)
+#  configure_file( DataStringGlobals.in.cc "${CMAKE_CURRENT_BINARY_DIR}/DataStringGlobals.cc" )
+#  configure_file( ConfiguredFunctions.in.cc "${CMAKE_CURRENT_BINARY_DIR}/ConfiguredFunctions.cc" )
+#  file( WRITE "${CMAKE_CURRENT_BINARY_DIR}/EmbeddedEpJSONSchema.cc" "Will be filled in later by GenerateEmbeddedEpJSONSchema" )
+#endif()
+
+set( HDRS include/AirflowNetwork/Solver.hpp
+          include/AirflowNetwork/Elements.hpp
+          include/AirflowNetwork/Properties.hpp
+)
+
+set( SRCS src/Solver.cpp
+          src/Elements.cpp
+          src/Properties.cpp
+)
+
+#include_directories( include )
+
+#RoomAirModelAirflowNetwork.cc
+#RoomAirModelAirflowNetwork.hh
+#RoomAirModelManager.cc
+#RoomAirModelManager.hh
+#RoomAirModelUserTempPattern.cc
+#RoomAirModelUserTempPattern.hh
+
+#CREATE_SRC_GROUPS( "${SRC}" )
+
+if (UNIX)
+  ADD_DEFINITIONS("-fPIC")
+endif()
+
+# Create a static library
+# this will be linked statically to create the DLL and also the unit tests
+add_library( airflownetworklib STATIC ${SRCS} ${HDRS} )
+
+target_include_directories( airflownetworklib PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include )
```

Comment:
Once all the comments are cleaned out of here, this looks good to go.  I like that it is a little standalone static lib now.